gem/resources/js/Extensions.js

```diff
@@ -972,7 +972,7 @@ extendPrototype(UIAElement, {
             var vectors = [];
 
             // iOS 8.1 takes between 3 and 5 milliseconds each (????!?!?!) to evaluate these, so only do it for 7.x
```

Comment:
Should add a comment that we force the fully-annotated output when we are logging errors.